extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialogModel.ts

```diff
@@ -20,6 +20,8 @@ export class MigrationCutoverDialogModel {
 			this._migration.subscription,
 			this._migration.migrationContext
 		));
+		// Printing migration status for private preview so that users can report their errors.
+		console.log(this.migrationStatus);
```

Comment:
1. Why not use an output channel for this?
2. You can probably remove the comment - or at least just have it say that this is to help debugging
3. When printing stuff to the console please always make sure to add text giving context - just having a string show up in the logs like this is going to be confusing if people don't know what this is from